server/test/integration/dbTest/beginTransactionTest.js

```diff
@@ -85,7 +85,10 @@ describe('db.js integration tests: beginTransaction', () => {
 
   describe('when beginTransaction fails twice, deadlocking on the first attempt and throwing an error with a valid pgClient on the second attempt', () => {
     let clientStub
+    let rollbackTransactionStub
     beforeEach(async () => {
+      rollbackTransactionStub = sandbox.stub()
+      db.__set__('rollbackTransaction', rollbackTransactionStub)
```

Comment:
Because `rollbackTransaction` is exported by `db.js`, I think that you can use `sandbox.stub(db, 'rollbackTransaction') instead of `rewire`.

Reply:
After looking into this, stubbing rollbackTransaction doesn't seem to be working, and we think perhaps the rewire on line 11 is causing this `const db = rewire('../../../db/db')`. Because we still need to use the rewire for other reasons, I will keep this code as is.